x-pack/plugins/security_solution/cypress/objects/login_url.ts

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
```

Comment:
Is this a leftover/placeholder, or necessary stub for cypress?

Reply:
leftovers 😬 I'll delete it.